### PR TITLE
UI Component My Weight Tile

### PR DESCRIPTION
**Pull request by skorpiom**
Hey,

This component is displayed in overview board.

You can start it using: 
`npm run start:tileMyWeight`

I imported Montserrat font in App.css file, is it ok?

### CODE REVIEW

File: Src/UIComponents/TileMyWeight/TileMyWeight.ts
Comment (fszwedo) [inline Src/UIComponents/TileMyWeight/TileMyWeight.ts:2]: isn't typescript throwing some warnings for 'require' here? I had some issues with that :)

**Review by fszwedo — APPROVED** [review]

**Comment by fszwedo** [comment]
@skorpiom there are some conflicts to be resolved, apart from that I don't see any issues

**Review by robert-czwartosz — APPROVED** [review]

**Comment by fszwedo** [comment]
@skorpiom I guess you can merge this one after the conflicts will be resolved, both approvals are in place

File: Src/UIComponents/TileMyWeight/TileMyWeight.ts
Comment (skorpiom) [reply Src/UIComponents/TileMyWeight/TileMyWeight.ts:2]: Yea I'm trying to change it into import but there are some other issues.

**Comment by skorpiom** [comment]
@fszwedo @robert-czwartosz 
I've got some problems with moment.js library - changed 'require' into 'import' and it works fine if I use parcel to generate unit implementation but I've got a problem with tests locally and here on github. 

Can I merge or should I change it back into 'require' ?

**Comment by fszwedo** [comment]
from my side I'd go back to require as it sounds like the least time-consuming option and using that it's no 'threat' for the project, it works just fine right?

**Comment by skorpiom** [comment]
Yea, with require parcel builds everything well and tests on github passes.

**Pull request merged**
